core/src/test/java/org/infinispan/container/SimpleDataContainerTest.java

```diff
@@ -59,38 +66,39 @@ public void testExpiredData() throws InterruptedException {
       InternalCacheEntry entry = dc.get("k");
       assertNotNull(entry);
       assertEquals(transienttype(), entry.getClass());
-      assertTrue(entry.getLastUsed() <= System.currentTimeMillis());
+      assertTrue(entry.getLastUsed() <= timeService.wallClockTime());
       long entryLastUsed = entry.getLastUsed();
-      Thread.sleep(100);
+      timeService.advance(100);
       entry = dc.get("k");
       assertTrue(entry.getLastUsed() > entryLastUsed);
       dc.put("k", "v", new EmbeddedMetadata.Builder().maxIdle(0, TimeUnit.MINUTES).build());
 
       dc.put("k", "v", new EmbeddedMetadata.Builder().lifespan(100, TimeUnit.MINUTES).build());
-      Thread.sleep(100);
+      timeService.advance(100);
       assertEquals(1, dc.size());
 
       entry = dc.get("k");
       assertNotNull(entry);
       assertEquals(mortaltype(), entry.getClass());
-      assertTrue(entry.getCreated() <= System.currentTimeMillis());
+      assertTrue(entry.getCreated() <= timeService.wallClockTime());
```

Comment:
Sure. I didn't think to go back and change all of these. This test was so outdated :P